client/.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 logs
 *.logs.env
 *.env
+.idea
```

Comment:
you still have idea in the repo

Reply:
![Screenshot 2022-07-13 at 10 25 36](https://user-images.githubusercontent.com/71763659/178687132-a01f0db9-a707-435d-b76f-da966b53dca0.png)